### DISCUSSION TITLE
Infinite levels due to mindwipe (captain's "Family" talent)

### DISCUSSION THREAD

**Discussion by Banditism** [FakeFishGames/Barotrauma#14988] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

if you use a mindwipe and a captain with atleast 8 skill points you can make a net neutral of skill points but positive for the rest of the crew, already stupid enough. after doing this for awhile i realized if you dont spend all your skills and leave the last one out before wiping again you actually dont lose a single skill.... so basically you can use a mindwipe and give everyone a free level including yourself..... this is really op and insanely hilarious

### Reproduction steps

captain with 8 skill points, tech into family
dont spend last point and mindwipe
captain with 9 skill points tech into family 
dont spend last point and mindwipe
captain with 10 skill points tech into family 
dont spend last point and mindwipe
captain with 11 skill points tech into family 
dont spend last point and mindwipe
captain with 12 skill points tech into family 
dont spend last point and mindwipe
captain with 13 skill points tech into family 
dont spend last point and mindwipe

you get the idea....

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.18.1 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Banditism** [comment]
after noticing my game i realized mindwipe broke my characters sync even harder, i keep my current skills but my xp gains got reset as if i was a fresh character
![image](https://github.com/user-attachments/assets/fb3e94e7-28b8-47bf-b097-b946e731af61)
note mainly the xp to get next level

**Comment by Regalis11** [comment]
This has been addressed in our private repo now: each successive use of mindwipe increases the talent point penalty by one. That still means you can abuse the Family talent a couple of times, but it becomes more and more expensive every time you do it. Fix coming up in the next unstable build and the upcoming release.
